frontend/src/ui/InitialConsent.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useRef, useState } from "react";
+import CONFIG from "../config";
+import { currentRef, useTranslatedConfig } from "../util";
+import { Modal, ModalHandle } from "./Modal";
+import { Button } from "./Button";
+import { TextBlock } from "./Blocks/Text";
+
+
+const USER_CONSENT = "tobiraUserConsent";
```

Comment:
```suggestion
const LOCAL_STORAGE_KEY = "tobiraUserConsent";
```

I think that's more fitting: the variable does not contain the "user consent". And since it's not exported from this module, it doesn't even need to contain "user consent" in the name.